PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskPID.cxx

```diff
@@ -1277,23 +1277,23 @@ void AliAnalysisTaskPID::UserCreateOutputObjects()
     fTPCclusterStudies = new TObjArray*[arrNumbers];
     const Int_t histNumbers = 3;
     
-    const Int_t nBinsY[histNumbers+1] = {8, 4, 11, 15};
+    constexpr const Int_t nBinsY[histNumbers+1] = {8, 4, 11, 15};
     Float_t twopi = 2.0 * TMath::Pi();
     
     Double_t binsPrime[nBinsPrime+1];
     for (Int_t i=0;i<=nBinsPrime;i++) {
       binsPrime[i] = primeLow + i*(primeHigh - primeLow)/nBinsPrime;
     }
     
-    Double_t binsYEta[nBinsY[0] + 1] = {-0.9, -0.6, -0.4, -0.2, 0.0, 0.2, 0.4, 0.6, 0.9};
-    Double_t binsYPhi[nBinsY[1] + 1] = {0.0, twopi/4.0, twopi/2.0, 3.0 * twopi/4.0, twopi};
-    Double_t binsYMult[nBinsY[2] + 1] = {0, 100, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000};
+    constexpr Double_t binsYEta[nBinsY[0] + 1] = {-0.9, -0.6, -0.4, -0.2, 0.0, 0.2, 0.4, 0.6, 0.9};
+    constexpr Double_t binsYPhi[nBinsY[1] + 1] = {0.0, twopi/4.0, twopi/2.0, 3.0 * twopi/4.0, twopi};
+    constexpr Double_t binsYMult[nBinsY[2] + 1] = {0, 100, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000};
```

Comment:
Here const was sufficient but constexpr doesn't hurt - constexpr is actually more general and therefore in c++11 code often the better choice.